.stylelintrc.json

```diff
@@ -0,0 +1,124 @@
+{
+  "rules": {
+    "at-rule-empty-line-before": [ "always", {
+      "except": [ "blockless-group", "first-nested" ],
+      "ignore": ["after-comment"]
+    }],
+    "at-rule-name-case": "lower",
+    "at-rule-name-space-after": "always-single-line",
+    "at-rule-semicolon-newline-after": "always",
+    "block-closing-brace-newline-after": "always",
+    "block-closing-brace-newline-before": "always",
+    "block-no-empty": true,
+    "block-opening-brace-newline-after": "always",
+    "block-opening-brace-space-before": "always",
+    "color-hex-case": "lower",
+    "color-hex-length": "short",
+    "color-no-invalid-hex": true,
+    "declaration-bang-space-after": "never",
+    "declaration-bang-space-before": "always",
+    "declaration-block-no-ignored-properties": true,
+    "declaration-block-no-shorthand-property-overrides": true,
+    "declaration-block-properties-order": [[
+    {
+      "order": "flexible",
+      "properties": [
+        "position",
+        "z-index",
+        "top",
+        "right",
+        "bottom",
+        "left"
+      ]},
+    {
+      "order": "flexible",
+      "properties": [
+        "height",
+        "max-height",
+        "min-height",
+        "width",
+        "max-width",
+        "min-width",
+        "display",
+        "box-sizing",
+        "overflow",
+        "overflow-x",
+        "overflow-y",
+        "padding",
+        "padding-top",
+        "padding-right",
+        "padding-bottom",
+        "padding-left",
+        "border",
+        "border-top",
+        "border-right",
+        "border-bottom",
+        "border-left",
+        "border-radius",
+        "margin",
+        "margin-top",
+        "margin-right",
+        "margin-bottom",
+        "margin-left"
+      ]}], { "unspecified": "bottom" }],
```

Comment:
<img width="621" alt="screen shot 2016-05-13 at 3 41 46 pm" src="https://cloud.githubusercontent.com/assets/10803178/15259762/32c8fd9e-1921-11e6-8a15-af6c8ceab81a.png">
I concede, it is smarter than me and knows about prefixes...
also, there doesn't seem to be any cases where the more specific attr would come before the general one - so I will make it alphabetical